core/src/main/java/org/elasticsearch/action/support/WriteRequest.java

```diff
@@ -65,45 +65,70 @@ default R setRefreshPolicy(String refreshPolicy) {
         /**
          * Don't refresh after this request. The default.
          */
-        NONE,
+        NONE((byte) 0, Boolean.FALSE.toString()),
         /**
          * Force a refresh as part of this request. This refresh policy does not scale for high indexing or search throughput but is useful
          * to present a consistent view to for indices with very low traffic. And it is wonderful for tests!
          */
-        IMMEDIATE,
+        IMMEDIATE((byte) 1, Boolean.TRUE.toString()),
         /**
          * Leave this request open until a refresh has made the contents of this request visible to search. This refresh policy is
          * compatible with high indexing and search throughput but it causes the request to wait to reply until a refresh occurs.
          */
-        WAIT_UNTIL;
+        WAIT_UNTIL((byte) 2, "wait_for");
+
+        private final byte id;
```

Comment:
can we remove the id member and stick with the enum ordinal like we were doing before?